src/openforms/emails/utils.py

```diff
@@ -100,3 +103,23 @@ def render_email_template(
         backend=openforms_backend,
         disable_autoescape=disable_autoescape,
     )
+
+
+def collect_failed_emails(since: datetime) -> list[dict[str, str] | None]:
```

Comment:
I'd prefer a more specific type hint than just a dict. I'd even set this up as a dataclass:

```py
@dataclass
class FailedEmail:
    submission_uuid: uuid.UUID
    event: str  # or Literal?


def collect_failed_emails(since: datetime) -> Iterable[FailedEmail]:
    ...
```